src/Console/Command/Local.php

```diff
@@ -98,6 +98,20 @@ private function handlerFromServerlessYml(string $function, ?string $config): st
 
         $serverlessConfig = Yaml::parseFile($file, Yaml::PARSE_CUSTOM_TAGS);
 
+        if (isset($serverlessConfig['functions'])) {
```

Comment:
Could you add a comment that explains _why_ this block of code exists? That will greatly simplify maintenance in the future.